quick-build.sh

```diff
@@ -41,4 +41,4 @@ $$$$"""$$$$$$$$$$uuu   uu$$$$$$$$$"""$$$"
 EOF
 
 ./stop.sh
-mvn clean install -B -e -V -DskipTests=true -Dmaven.source.skip=true -Pdev,skipSanityChecks $*
+mvn clean install -B -e -V -Dmaven.test.skip -Dmaven.source.skip=true -Pdev,skipSanityChecks $*
```

Comment:
I was relying on the fact this script compiled the tests and just did not execute them.
It's convenient to refresh all local artifacts quickly in the morning.

I'll just stop using this script.

Reply:
Also, it's convenient to just get compiler check on all code before just pushing to CI to get full test and QA feeback.